localstack/services/awslambda/lambda_api.py

```diff
@@ -1753,7 +1753,7 @@ def put_function_event_invoke_config(function):
     response = lambda_obj.put_function_event_invoke_config(data)
 
     return jsonify({
-        'LastModified': timestamp_millis(response.last_modified),
+        'LastModified': response.last_modified.strftime('%Y-%m-%dT%H:%M:%S.%f') + '+00:00',
```

Comment:
Thanks for this PR @mgagliardo . 👍 

nitpick: Could we put the time format in a constant at the global scope in this file (or use an existing one from `utils/common.py`)? This would help us make the time format reusable, in case we may need it somewhere else in this file. 

Something like:
```
DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%f+00:00'
...
response.last_modified.strftime(DATE_FORMAT)
```

Reply:
Thanks for the comments, I spent some time testing every dateformat in utils.py and could not find any matching this one as it is very specific, just needs the TZ region. 
As it is the only one matching lambda I have put it here, but if you think it is better to put it in `utils/common.py` I have no problem with that.